### ISSUE TITLE
Members can't use Trader NPC.

### ISSUE THREAD

**Issue by hOlyYy** [CitizensDev/Citizens#748]
I'm using latest PermissionEx, user is in group Member which has those perms:
     permissions:
        - modifyworld.*
        - openwarp.*
        - iConomy.help
        - iConomy.holdings
        - iConomy.payment
        - iConomy.holdings.others
        - residence.create
        - citizens.use.*
        - citizens.basic.use.info
When they rightclick nothing happens.
Or when ops try to buy money is not taken from their accounts.
iConomy  6.0.8b.

**Comment by kripts** [comment]
Use this permission to allow them to trade: 
citizens.trader.use 
And for the money not been taken from the account, you need to disable ChatManager, it seems to conflict with Citizens.

**Comment by hOlyYy** [comment]
Yes kripts i found bug in permission it was supposed to be citizens.trader.use not use.trader.
Everything is working fine. ATM. even with ChatManager Running.
